File: app/src/main/java/com/abeltran10/carajilloapp/MainActivity.java
package com.abeltran10.carajilloapp;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import com.abeltran10.carajilloapp.ui.login.LoginFragment;
import com.abeltran10.carajilloapp.ui.main.MainFragment;
import com.google.firebase.auth.FirebaseAuth;


public class MainActivity extends AppCompatActivity {
    private FirebaseAuth mAuth = FirebaseAuth.getInstance();
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_main);
        Toolbar toolbar = findViewById(R.id.my_toolbar);
        setSupportActionBar(toolbar);
        getSupportActionBar().setLogo(R.mipmap.ic_launcher_round);

        if (savedInstanceState == null)
            if (mAuth.getCurrentUser() != null) {
                getSupportFragmentManager().beginTransaction().setReorderingAllowed(true)
                        .add(R.id.frame_container, MainFragment.class, null)
                        .commit();
            } else {
                getSupportFragmentManager().beginTransaction().setReorderingAllowed(true)
                        .add(R.id.frame_container, LoginFragment.class, null)
                        .commit();
            }


    }

}

File: app/src/main/java/com/abeltran10/carajilloapp/MainActivity.kt
package com.abeltran10.carajilloapp

import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.appcompat.app.AppCompatActivity
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Surface
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.tooling.preview.Preview
import com.abeltran10.carajilloapp.ui.login.LoginFragment
import com.abeltran10.carajilloapp.ui.theme.CarajilloAppTheme

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.fragment_login)
    }
}

//@Composable
//fun Greeting(name: String, modifier: Modifier = Modifier) {
//    Text(
//        text = "Hello $name!",
//        modifier = modifier
//    )
//}
//
//@Preview(showBackground = true)
//@Composable
//fun GreetingPreview() {
//    CarajilloAppTheme {
//        Greeting("Android")
//    }
//}